azure-storage-blob-source.kamelet.yaml

```diff
@@ -0,0 +1,60 @@
+    camel.apache.org/provider: "Apache Software Foundation"
+  labels:
+    camel.apache.org/kamelet.type: "source"
+spec:
+  definition:
+    title: "Azure Storage Blob Source"
+    description: |-
+      Consume Files from Azure Storage Blob.
+    required:
+      - period
+      - accountName
+      - containerName
+      - accessKey
+    properties:
+      period:
+        title: Period between polls
+        description: The interval between fetches to the Azure Storage Container in milliseconds
+        type: integer
+        default: 10000
+      accountName:
+        title: Account Name
+        description: The Azure Storage Blob account name.
+        type: string
+      containerName:
+        title: Container Name
+        description: The Azure Storage Blob container name.
+        type: string
+      accessKey:
+        title: Access Key
+        description: The Azure Storage Blob access Key.
+        type: string
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:password
+  flow:
+    from:
+      uri: "timer:azure-storage-blob-stream"
+      parameters:
+        period: "{{period}}"
+      steps:
+      - to:
+          uri: "azure-storage-blob:{{accountName}}/{{containerName}}"
+          parameters:
+            operation: "listBlobs"
+            accessKey: "{{accessKey}}"
+      - split:
+          jsonpath: "$.*"
+          steps:
+          - set-property:
+              name: azureBlobName
+              simple: ${body.name}
+          - tod: "azure-storage-blob:{{accountName}}/{{containerName}}?accessKey=RAW({{accessKey}})&operation=getBlob&blobName=${exchangeProperty.azureBlobName}"
```

Comment:
Can you change them to "to-d" (see https://issues.apache.org/jira/browse/CAMEL-16490)